src/_variables.scss

```diff
@@ -1,3 +1,8 @@
 $primary : #00897b;
 $primary-light : #4ebaaa;
-$primary-dark : #005b4f;
\ No newline at end of file
+$primary-dark : #005b4f;
+
+$white: #fcfcfc;
+
```

Comment:
I like the naming conventions on this, I definitely need to get better. way to go